### PR TITLE
Use static-build of LWS library created by installer

### LINKED ISSUE #8152: ACLK update build to use libwebsocket static lib
##### Feature idea summary
On systems with Libwebsockets <3 we should use Libwebsockets 3 build by installer.

##### Expected behavior
autotools build system should (in following order):
- Check shared library libwebsockets 3+ is present in system. If it is use it for ACLK
- Check static library build of libwebsockets exists (by installer). If it is use it for ACLK
- If both previous steps failed do not build ACLK

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
~~If shared library libwebsockets > 3.0.0 not present in system try if static version was built by installer. If yes use static lib otherwise dont build ACLK.~~

Update:
After discussion it has been decided to only use our custom build of LWS (by installer) or disable ACLK. This PR has been updated to mirror that decision.

Closes #8152

### CODE REVIEW

File: packaging/installer/methods/manual.md [hunk 1]
```diff
@@ -115,6 +115,16 @@ Netdata DB engine can be enabled when these are installed (they are optional):
 
 *Netdata will greatly benefit if you have the above packages installed, but it will still work without them.*
 
+Netdata Cloud support may require the following packages to be installed:
+
+| package  | description
+|:--------:| -----------------------
+| `libwebsockets-devel` } Version 3 or higher is at needed build time for websockets support for Netdata Cloud
```
Comment (shortpatti) [inline packaging/installer/methods/manual.md]:
```suggestion
| `libwebsockets-devel` | Version 3 or higher is at needed build time for websockets support for Netdata Cloud
```

Comment (Ferroin) [reply packaging/installer/methods/manual.md]: Resolved in upstream code (this came from #8144 which this PR is based on).

**Comment by Ferroin** [comment]
@underhood At this point I think the title needs updated on this so that it's correct in the changelog for the next release.

**Comment by amoss** [comment]
Fails to build when I try it on Debian 10. Please add details of how you tested this.

**Comment by amoss** [comment]
Please rebase branch on master before requesting review.

**Comment by underhood** [comment]
at the time of PR the @Ferroin PR was not merged yet so I kept it like this for easier testing. I rebased it. I go on to check the debian thing.

**Comment by underhood** [comment]
seems to be #8144 related I will continue check this.

**Comment by amoss** [comment]
This looks correct but it cannot be approved until the installer is fixed to overcome the CLFAGs issue (and anything to do with libcap) otherwise it will break access to the ACLK in our current tree.

**Pull request merged**